aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogTable.java

```diff
@@ -269,14 +269,6 @@ public void testRenameTableFailsToDeleteOldTable() {
             .databaseName(namespace)
             .tableInput(TableInput.builder().name(tableName).parameters(Maps.newHashMap()).build())
             .build());
-    AssertHelpers.assertThrows(
-        "should fail to rename",
-        ValidationException.class,
-        "Input Glue table is not an iceberg table",
-        () ->
-            glueCatalog.renameTable(
-                TableIdentifier.of(namespace, tableName),
-                TableIdentifier.of(namespace, newTableName)));
```

Comment:
Why is this assertion removed? It looks like it's for rename

Reply:
This assertion was removed due to the logic change in renameTable that allows rename table to use the previous table's Iceberg Properties (metadata location) the related integration test would always fail at this assert.